### DISCUSSION TITLE
DAI 2024-2025 - My Docker Compose application - Kenan Augsburger

### DISCUSSION THREAD

**Discussion by Mondotosz** [heig-vd-dai-course/heig-vd-dai-course#409] [Show and tell]
Here's the link to the pull request which includes the content for the practical.
[Thynkon/dai-lab-01](https://github.com/Thynkon/dai-lab-01/pull/9)

(The compose is quite impractical in this case...)

**Comment by ludelafo** [comment]
Thank you!

You can always find the solution in the solution repository if you ever need to compare your work with the official one here: https://github.com/heig-vd-dai-course/heig-vd-dai-course-solutions.

Do not hesitate to ask questions if something is not clear, either in this discussion or in the Q&A category of the GitHub Discussions here: https://github.com/orgs/heig-vd-dai-course/discussions/categories/q-a.
